src/separations_tests.cc

```diff
@@ -42,7 +42,153 @@ TEST(SeparationsTests, SepMaterial) {
   EXPECT_DOUBLE_EQ(effs[id("Am241")] * mqorig.mass("Am241"), mqsep.mass("Am241"));
   EXPECT_DOUBLE_EQ(0, mqsep.mass("Am242"));
 }
+  
+TEST(SeparationsTests, SeparationEfficiency) {
```

Comment:
Nevermind - this checks 1.0 eff works - but please put a doc comment explaining exactly what this test is checking for.